receiver/nginxreceiver/metadata.yaml

```diff
@@ -1,6 +1,8 @@
 name: nginxreceiver
 
 labels:
+  state:
+    description: The state of a connection
 
```

Comment:
Looks like this can be an enum: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/metric-metadata.yaml#L14